spec/namespaces/security.yaml

```diff
@@ -1920,7 +2170,7 @@ components:
       in: query
       description: The domain names from all certificates.
```

Comment:
Description seems wrong, the one from `schema` should replace it. Check if there are others like this.

Reply:
updated all parameter descriptions